.pre-commit-config.yaml

```diff
@@ -59,8 +59,9 @@ repos:
       name: pyright
       entry: pyright
       language: node
+      files: ^src/
       types: [ python ]
-      pass_filenames: true
+      require_serial: true
```

Comment:
`pass_filenames` is the default, pyright uses multiprocessing so `require_serial` prevents a forkbomb

Reply:
Does this potentially cause pyright to take longer to run?

Reply:
it's ~slightly faster now:
- before there was resource contention due to pre-commit's multiprocessing + pyright's multiprocessing leading to exhaustion
- after pyright is now properly following imports so it does a little more work

`--all-files` goes from 12.7s to 12.6s in my tests